### ISSUE TITLE
Body of response is missing

### ISSUE THREAD

**Issue by SandeepAggarwal** [netguru/ResponseDetective#23]
In 'develop' branch response 'Body' is none

**Comment by akashivskyy** [comment]
That is too little information. I cannot reproduce it.

**Comment by SandeepAggarwal** [comment]
Sorry about that, I am using following code to set up ResponseDetective.

    NSURLSessionConfiguration *configuration = [NSURLSessionConfiguration defaultSessionConfiguration];
    [RDTResponseDetective enableInConfiguration:configuration];

**Comment by akashivskyy** [comment]
For which request/response the body is missing?

**Comment by SandeepAggarwal** [comment]
GET request. Response body is missing

**Comment by akashivskyy** [comment]
I'm sorry, I still cannot reproduce it for `GET https://httpbin.org/get`.

**Comment by SandeepAggarwal** [comment]
```
    NSURLSessionConfiguration *configuration = [NSURLSessionConfiguration defaultSessionConfiguration];
    [RDTResponseDetective enableInConfiguration:configuration];
 id<TRCConnection> connection = [[TRCConnectionNSURLSession alloc] initWithBaseUrl:[NSURL URLWithString:@"SomeURL"] configuration:configuration];
```

I am using [Typhoon Rest client ](https://github.com/appsquickly/TyphoonRestClient). May be this will give you any hint.

**Comment by akashivskyy** [comment]
It doesn't unfortunately. One last question: What are the `Content-Type` and `Content-Length` headers of the response logged by ResponseDetective?

**Comment by SandeepAggarwal** [comment]
Content-Type: application/json; charset=utf-8
 Content-Length: 19099

**Comment by akashivskyy** [comment]
Ok, I identified the issue. It seems that ResponseDetective does not honor a valid `Content-Type` headers containing parameters (such as `; charset=utf-8` in your case).

This bug will be tracked in #24 from now on.

Thanks for reporting! 🙇

**Comment by SandeepAggarwal** [comment]
Glad, it helped you track a bug. Btw, How much early can I expect to get it fixed?

**Comment by akashivskyy** [comment]
Seems to be a fairly simple thing. I expect to fix this and release a stable version 1.0 by the end of this week. 😉
